lib/probe/http-client.js

```diff
@@ -247,10 +247,19 @@ class HttpRequestMetrics {
         // ...otherwise count actual response size as its streamed
         if (isNaN(this.responseSize)) {
             this.responseSize = 0;
+
+            // prevent stream from being read as soon as we add our 'data' listener below,
+            // leave that to the actual client code consuming the request
+            response.readableFlowing = false;
```

Comment:
Does this need a documentation update then?

Reply:
No. This doesn't change expected behavior, it fixes a bug where expected behavior of client code was broken.